### DISCUSSION TITLE
flex order dependent on screen size

### DISCUSSION THREAD

**Discussion by nielsswinkels** [quasarframework/quasar#15979] [General - Components / Directives / etc]
Currently you can do responsive design by changing the size of flex cells like this
`col-xs-12 col-sm-6 col-md-4`
And you can change the order with classes like this
`order-first`

But as far as I can find there is no (quasar built-in) way to change the order dependent on the screen size? Wouldn't it be nice with classes like
`order-xs-first` ?

**Comment by metalsadman** [comment]
use screen helper https://quasar.dev/options/screen-plugin#screen-api
